app/views/fines/_payment.html.erb

```diff
@@ -1,19 +1,19 @@
 <li class="list-group-item">
   <div class="row d-flex flex-wrap position-relative">
     <div class="d-flex flex-row flex-grow-1 justify-content-start w-100 row col-md-12 col-lg-6">
-      <div class="col-4 col-md-4 order-md-first bill_description" data-sort-bill_description="<%= h payment.sort_options(:bill_description) %>" >
+      <div class="col-4 col-md-4 order-md-first bill_description" data-sort-bill_description="<%= h payment.sort_key(:bill_description) %>" >
```

Comment:
should this be `nice_bill_description` instead?

Reply:
Not here, but, sure, in the model, probably.